autogpt/config/ai_config.py

```diff
@@ -16,9 +15,6 @@
     from autogpt.models.command_registry import CommandRegistry
     from autogpt.prompts.generator import PromptGenerator
 
-# Soon this will go in a folder where it remembers more stuff about the run(s)
-SAVE_FILE = str(Path(os.getcwd()) / "ai_settings.yaml")
```

Comment:
Good, I hate that this is distributed everywhere (and the implementations differ)